server/models/Conversation.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require("mongoose");
+
+const messageSchema = new mongoose.Schema(
+  {
+    sender: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    receiver: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+      ref: "User",
+    },
+    description: {
+      type: String,
+      default: "",
+    },
+  },
+  { timestamps: true }
+);
+
+const conversationSchema = new mongoose.Schema(
+  {
+    messages: {
+      type: [messageSchema],
```

Comment:
Why don't we store a refs to the Message model and populate on demand? :)

Reply:
ok that seems better